scripts/constants.js

```diff
@@ -9,6 +9,10 @@ const submitFailStyle = 'background-color: #e32636; color: white; padding: 4px 8
 const difficulty_colors = ['green', 'rgb(239, 108, 0)', 'red'];
 
 const replacements = {
+  ":biggwiggle:": "https://cdn.discordapp.com/emojis/1022965811498979378.gif?size=96&quality=lossless",
+  ":bigwiggle:": "https://cdn.discordapp.com/emojis/1022965811498979378.gif?size=96&quality=lossless",
+  ":thonk:": "https://cdn.discordapp.com/emojis/539305279217598474.webp?size=96&quality=lossless",
+  ":cope:": "https://cdn.discordapp.com/emojis/752324497935171664.webp?size=96&quality=lossless",
```

Comment:
![image](https://user-images.githubusercontent.com/19920441/198356696-d43b00fc-cde7-4544-b877-d3868ff68651.png)

Reply:
Great point @sunnyguan! I'll make that change immediately. Sorry for the delay.